lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -105,10 +166,7 @@
                                 passwordResetSupportUrl: passwordResetSupportUrl,
                                 linkTitle: gettext('Reset Your Password'),
                                 linkHref: fieldsData.password.url,
-                                helpMessage: StringUtils.interpolate(
-                                    gettext('When you select "Reset Your Password", a message will be sent to the email address for your {platform_name} account. Click the link in the message to reset your password.'),  // eslint-disable-line max-len
-                                    {platform_name: platformName}
-                                )
+                                helpMessage: gettext('Check your email account for instructions to reset your password.')  // eslint-disable-line max-len
```

Comment:
just a question, can we point out the user current email id in that message?

Reply:
the email address will be visible in its related email field.

Reply:
may be we can take suggestion from product owner here?

Reply:
This help text will be displayed on account settings pages where a learner can view all field including the email address. Please check sandbox https://business.sandbox.edx.org/account/settings.

Reply:
ok got it. Thanks